hackathon_site/event/views.py

```diff
@@ -200,3 +208,36 @@ def post(self, request, *args, **kwargs):
         at once.
         """
         return super().post(request, *args, **kwargs)
+
+
+class LeaveTeamView(generics.GenericAPIView):
```

Comment:
Because this doesn't have a `serializer_class` property or `get_serializer_class` method, the endpoint will crash (500) if you try to open it in the browser. 

Crashing isn't good, so you could do `serializer_class = TeamSerializer`. Then if you submit a get request, you'll get a 405 - Method not Allowed back.

That will make your `swagger_auto_schema` unnecessary, since the 201 will be annotated properly. The downside is that DRF / Swagger thinks the endpoint needs a team code to be in the request body, which isn't the case. But the API still works fine without it.

Reply:
This should be in `event/api_views.py`